### PR TITLE
CICE-15 Apply filters

### PR DESCRIPTION
**Pull request by GonzaEnrique27**
Apply filters for the different search categories.

Provide a category display option in the upper right corner relative to the results grid.

### CODE REVIEW

File: pages/places/_id/place-details.vue [hunk 1]
```diff
@@ -33,10 +33,29 @@
         {{ wikiExtract != '' ? wikiExtract : 'No hay descripción' }}
```
Comment (njrzr) [inline pages/places/_id/place-details.vue:33]: @GonzaEnrique27 there are conflict with this file, fix it. If you need help fixing this send a message on slack.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "@nuxtjs/pwa": "^3.3.5",
```
Comment (njrzr) [inline package.json:16]: @GonzaEnrique27 there's a conflict here too.

File: components/WLCloseByMiniatures.vue [hunk 3]
```diff
@@ -146,6 +153,141 @@ export default {
...
+          break
+      }
+      const url = wbk.sparqlQuery(query)
+      fetch(url, {
+        method: 'GET'
+      })
+        .then(res => res.json())
+        .then((data) => {
+          this.locations = data.results.bindings
+          console.log(this.locations)
```
Comment (njrzr) [inline components/WLCloseByMiniatures.vue:286]: @GonzaEnrique27 delete this console.log

File: components/WLCloseByMiniatures.vue [hunk 4]
```diff
@@ -146,6 +153,141 @@ export default {
...
+          this.$store.commit('setLocations', data.results.bindings)
+        })
+      console.log(query)
```
Comment (njrzr) [inline components/WLCloseByMiniatures.vue:289]: @GonzaEnrique27 delete this console.log

File: components/WLCloseByMiniatures.vue [hunk 5]
```diff
@@ -174,15 +315,14 @@ export default {
...
-
         const url = wbk.sparqlQuery(query)
-
         fetch(url, {
           method: 'GET'
         })
           .then(res => res.json())
           .then((data) => {
             this.locations = data.results.bindings
+            console.log(data.results.bindings)
```
Comment (njrzr) [inline components/WLCloseByMiniatures.vue:325]: @GonzaEnrique27 delete this console.log

**Review by njrzr — CHANGES_REQUESTED** [review]
@GonzaEnrique27 read the comments and fix these issues, with the merge conflict issue, as I wrote, if you need help send me a message.

**Review by njrzr — APPROVED** [review]

File: pages/places/_id/place-details.vue (hunk 1 above)
Comment (GonzaEnrique27) [reply pages/places/_id/place-details.vue]: fixed!

File: components/WLCloseByMiniatures.vue (hunk 3 above)
Comment (GonzaEnrique27) [reply components/WLCloseByMiniatures.vue]: fixed!

File: components/WLCloseByMiniatures.vue (hunk 5 above)
Comment (GonzaEnrique27) [reply components/WLCloseByMiniatures.vue]: fixed!

File: package.json (hunk 2 above)
Comment (GonzaEnrique27) [reply package.json]: fixed

File: components/WLCloseByMiniatures.vue (hunk 4 above)
Comment (GonzaEnrique27) [reply components/WLCloseByMiniatures.vue]: fixed!

**Pull request merged**